main/vstring.c

```diff
@@ -244,11 +244,7 @@ extern void vStringCopyToLower (vString *const dest, const vString *const src)
 		vStringResize (dest, src->size);
 	d = dest->buffer;
 	for (i = 0  ;  i < length  ;  ++i)
-	{
-		int c = s [i];
-
-		d [i] = tolower (c);
-	}
+		d [i] = (char) tolower (s [i]);
```

Comment:
Your change itself looks good to me.

Following the consideration that I made during reviewing is not directly related to this pull request. If I wrote incorrect things, please, let me know.

`s [i]` can be a negative if `char` on this platform means `signed char`.
In `tolower(3)` man page:

        If c is neither an unsigned char value nor EOF, the behavior of these functions is undefined. 


So `tolower (s [i])` can be a trouble. Am I wrong?

If we declare s as `unsigned char *`, and d as `unsigned char *`, 
we can avoid the trouble.